iree/hal/metal/CMakeLists.txt

```diff
@@ -0,0 +1,125 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if(NOT ${IREE_HAL_DRIVER_METAL})
+  return()
+endif()
```

Comment:
Ah, no `BUILD` file yet. I guess this is why you wanted to chat about Bazel conditional compilation? There are a few tricks we can use to keep developer and CI builds running across the project while having platform-specific directories. Just not including a `BUILD` file is simplest though... at least until this gets referenced by more parts of the project.

Reply:
Yup.